crates/recursor/src/recursor.rs

```diff
@@ -111,6 +112,13 @@ impl RecursorBuilder {
         self
     }
 
+    /// Sets whether to randomize the case of letters in query names, and require that repsonses
```

Comment:
For rustdoc it's usually better to have a single sentence that fits on a line. Then an empty line after that, then all the additional detail.

Also, "repsonses" is probably not what you meant.